io/io/inc/TFile.h

```diff
@@ -113,6 +113,8 @@ class TFile : public TDirectoryFile {
    TList           *fInfoCache{nullptr};      ///<!Cached list of the streamer infos in this file
    TList           *fOpenPhases{nullptr};     ///<!Time info about open phases
 
+   bool fLocallyManaged = false; ///<! if true, bypass use of global lists
```

Comment:
"Locally" could also mean "not remote"... Maybe a better name could be `fAvoidGlobalRegistration`?